schemas/store_location.json

```diff
@@ -0,0 +1,91 @@
+{
+  "title": "Store Location",
+  "type": "object",
+  "properties": {
+    "type": {
+      "type": "string",
+      "enum": [
+        "storeLocation"
+      ]
+    },
+    "id": {
+      "$ref": "types.json#/definitions/identifier"
+    },
+    "attributes": {
+      "type": "object",
+      "properties": {
+        "storeId": {
+          "type": "string"
+        },
+        "retailerName": {
+          "type": "string"
+        },
+        "phone": {
+          "type": "string",
+          "pattern": "^\\d{10}$"
+        },
+        "address": {
```

Comment:
it's better to use here a ref to address (if it's the same address we use everywhere)